gtsam/base/debug.h

```diff
@@ -47,6 +47,7 @@ namespace gtsam {
   // Non-guarded use led to crashes, and solved in commit cd35db2
   bool GTSAM_EXPORT guardedIsDebug(const std::string& s);
   void GTSAM_EXPORT guardedSetDebug(const std::string& s, const bool v);
+  bool GTSAM_EXPORT isDebugVersion();
```

Comment:
Separate from two other functions and add comment

Reply:
Done.